File: android-app/app/src/main/java/com/longmarch/game/data/GameData.kt
package com.longmarch.game.data

data class LevelInfo(
    val id: Int,
    val mapLabel: String,
    val shortDesc: String,
    val x: Float,
    val y: Float,
)

enum class LevelStatus { LOCKED, AVAILABLE, COMPLETED }

object GameData {
    const val CanvasWidth = 802f
    const val CanvasHeight = 433f

    // 与前端 mapData.ts / Windows 版本保持一致（802x433 逻辑画布坐标）
    val levels: List<LevelInfo> = listOf(
        LevelInfo(1, "瑞金", "伟大征程开始", 649f, 299f),
        LevelInfo(2, "湘江", "惨烈突围之战", 511f, 297f),
        LevelInfo(3, "遵义", "生死攸关转折", 403f, 271f),
        LevelInfo(4, "赤水", "用兵如神", 360f, 245f),
        LevelInfo(5, "金沙江", "跳出包围圈", 277f, 290f),
        LevelInfo(6, "泸定桥", "二十二勇士", 256f, 227f),
        LevelInfo(7, "雪山", "翻越夹金山", 285f, 211f),
        LevelInfo(8, "草地", "松潘大草地", 293f, 175f),
        LevelInfo(9, "腊子口", "攻克天险", 327f, 146f),
        LevelInfo(10, "吴起镇", "长征胜利！", 434f, 104f),
    )
}

File: windows-game/Data/GameData.cs
using LongMarchWindows.Models;

namespace LongMarchWindows.Data;

public static class GameData
{
    public const float CanvasWidth = 802f;
    public const float CanvasHeight = 433f;

    // 与前端 mapData.ts 保持一致（802x433 逻辑画布坐标）
    public static readonly IReadOnlyList<LevelInfo> Levels = new List<LevelInfo>
    {
        new() { Id = 1,  MapLabel = "瑞金",   ShortDesc = "伟大征程开始", X = 649, Y = 299 },
        new() { Id = 2,  MapLabel = "湘江",   ShortDesc = "惨烈突围之战", X = 511, Y = 297 },
        new() { Id = 3,  MapLabel = "遵义",   ShortDesc = "生死攸关转折", X = 403, Y = 271 },
        new() { Id = 4,  MapLabel = "赤水",   ShortDesc = "用兵如神",     X = 360, Y = 245 },
        new() { Id = 5,  MapLabel = "金沙江", ShortDesc = "跳出包围圈",   X = 277, Y = 290 },
        new() { Id = 6,  MapLabel = "泸定桥", ShortDesc = "二十二勇士",   X = 256, Y = 227 },
        new() { Id = 7,  MapLabel = "雪山",   ShortDesc = "翻越夹金山",   X = 285, Y = 211 },
        new() { Id = 8,  MapLabel = "草地",   ShortDesc = "松潘大草地",   X = 293, Y = 175 },
        new() { Id = 9,  MapLabel = "腊子口", ShortDesc = "攻克天险",     X = 327, Y = 146 },
        new() { Id = 10, MapLabel = "吴起镇", ShortDesc = "长征胜利！",   X = 434, Y = 104 },
    };
}
